callback_plugins/help.py

```diff
@@ -67,7 +156,8 @@ def v2_playbook_on_stats(self, stats):
                 self._display.banner("ERROR MESSAGE SUMMARY")
                 with io.open(FAIL_LOGFILE, 'r', encoding="utf-8") as f:
                     for _, line in enumerate(f):
```

Comment:
Use `for line in f:` is more elegant to read a file line by line(Still enumerated)